app/controllers/application_controller.rb

```diff
@@ -176,13 +197,6 @@ def set_sso_cookie!
     }
   end
 
-  def extend_session!
-    session.expire(Session.redis_namespace_ttl)
-    current_user&.identity&.expire(UserIdentity.redis_namespace_ttl)
-    current_user&.expire(User.redis_namespace_ttl)
-    set_sso_cookie!
-  end
-
   def destroy_sso_cookie!
     cookies.delete(Settings.sso.cookie_name, domain: Settings.sso.cookie_domain)
```

Comment:
are we ever destroying sso cookie outside of calling reset_session? maybe this could just live in that method.